udp_driver/config/udp_component_settings.yml

```diff
@@ -0,0 +1,4 @@
+udp_component:
+  ros__parameters:
+    ip: "192.168.10.5"
+    port: 57410
```

Comment:
In order to avoid having to change the node name in this parameters file when the node name is remapped, you can use "double-star" syntax:

```
/**:
  ros__parameters:
    ...
```

Reply:
whoa way cool i didnt know you could do that